cinnabar/measurements.py

```diff
@@ -52,19 +54,52 @@ class Measurement(DefaultModel):
 
     @classmethod
     def from_experiment(cls,
-                        pIC50: float,
+                        IC50: float,
                         label: str,
+                        uncertainty: float = 0,
                         source: str = '',
+                        temperature: float = 298,
```

Comment:
It's minor but I'd use 298.15 here instead, so we're consistent with everywhere else.